<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>644</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>644</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>791</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="ConnectionInfo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Information</string>
    </property>
    <widget class="QLineEdit" name="DomainEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="DomainLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Domain</string>
     </property>
    </widget>
    <widget class="QLabel" name="NameOfComputer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name of Computer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="RequestInterval">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Request Interval</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeoutValue">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Timeout Value</string>
     </property>
    </widget>
    <widget class="QSlider" name="TimeoutSlider">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="RequestSlider">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="RequestTimeBox">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>50</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="TimeoutTimeBox">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>110</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MySQLinfo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>MySQL Connection Information</string>
    </property>
    <widget class="QLabel" name="HostLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLabel" name="UserNameLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="DatabaseLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HostEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="DatabaseEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Additionalinfo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Additional Information and Alerts</string>
    </property>
    <widget class="QCheckBox" name="DisplayLogsCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Logs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AlertIfErrorCheckBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alert if error ocurs</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ApplyChangesButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopLoggingButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>520</y>
      <width>441</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Logging</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TimeDisplay">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="EmailBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Error Alerts</string>
    </property>
    <widget class="QLineEdit" name="EmailEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="EmailLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="CurrentTimeLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Time:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LogMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>431</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CopyrightEQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>© EquipoVision</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="SpecifyBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Specify Save Location For Log File</string>
    </property>
    <widget class="QLabel" name="LocationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Location: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LocationEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="NumFailuresLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Failures:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NetworkTesterLogoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/NetworkTesterLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumFailures">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
